modules/test/playwright/tests/portal-web/main/html/taglib/ui/inputLocalized.spec.ts

```diff
@@ -47,7 +49,39 @@ test(
 				.first()
 				.getAttribute('id');
 
-			expect(labelFor).toBe(inputId);
+			await expect(labelFor).toBe(inputId);
+		});
+	}
+);
+
+test(
+	'Input localized works although id includes some non alphanumeric characters',
+	{
+		tag: '@LPD-56164',
+	},
+	async ({page}) => {
+		await test.step('Check input localized is AUI compatible', async () => {
+			const labelFor = await page
+				.getByText('Sample label')
+				.getAttribute('for');
+
+			await expect(labelFor).toContain('inputLocalizedId_21_');
```

Comment:
This is the "encoding" we expect for `inputLocalizedId!`? I guess so but it's not immediate from the test

Reply:
Yes, that's it.